### ISSUE TITLE
Missing Semantics

### ISSUE THREAD

**Issue by michael-77** [scottcmoore/p-g-wodehouse_psmith-in-the-city#4]
In Chapter 7, `<i>Trifles</i>` should be marked up. I'm not sure exactly what it is - a fictional magazine perhaps? - and if you're not either then leave it as `<i>`.

In Chapter 7, `<i epub:type="se:name.vessel.ship">Lusitania,</i>` the comma should be outside the `<i>` tags

**Comment by scottcmoore** [comment]
I think you mean Chapter 19 for "Trifles". I was unable to find a record of a publication called Trifles -- every other publication I found in this book was a real London newspaper of the era, so I suspect that it's a real thing, but can't confirm that. I will leave it as `<i>` unless you think I should mark it up, maybe as something like `<i epub:type="se:name.publication">` ? Not sure if that's okay.

Moved the comma outside of the `<i>` in Chapter 7 in commit dd11a9f72ccccaa4e0211b852e88fa8508146923
